general-concepts/manifest/text.xml

```diff
@@ -7,68 +7,16 @@
 <title>Generating the Manifest</title>
 <body>
 <p>
-In the tree, every package has a <c>Manifest</c> file. This file lives in the same
-directory as the ebuilds for the package. The <c>Manifest</c> file contains digests
-(currently RMD160, SHA1, SHA256, SHA512 and WHIRLPOOL) and file size data for every
+In the tree, every package has a <c>Manifest</c> file. This file lives
+in the same directory as the ebuilds for the package. The <c>Manifest</c> file
+contains digests (currently BLAKE2B and SHA512) and file size data for every
```

Comment:
If it took us this long to update `currently RMD160, SHA1, SHA256, SHA512 and WHIRLPOOL` to `currently BLAKE2B and SHA512`, I'd say remove mentions to specific hash functions.